colorbleed/plugins/maya/create/colorbleed_yeti_cache.py

```diff
@@ -15,11 +15,12 @@ class CreateYetiCache(avalon.maya.Creator):
     def __init__(self, *args, **kwargs):
         super(CreateYetiCache, self).__init__(*args, **kwargs)
 
-        data = OrderedDict(self.data)
+        data = OrderedDict(**self.data)
```

Comment:
@aardschok Why did this need to change? Didn't the previous work or what happened? :)